manifests/pnp4nagios/config.pp

```diff
@@ -0,0 +1,77 @@
+class icinga2::pnp4nagios::config inherits icinga2::pnp4nagios::params {
+
+  $monitoring_type = 'icinga'
+  $log_type = 'syslog'
+  $debug_lvl = 1
+  $system_date = generate('/usr/bin/date', '+%Z')
+
+  file { "$httpd_path/$htpasswd_config":
```

Comment:
to resolve this, do take a look at how postgresql module is doing it https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/manifests/server/config.pp